### PR TITLE
[4.0] Media Manager update

### PR DESCRIPTION
**Pull request by laoneo**
The latest changes from the media manager repo. It's needed for the ES6 conversion repo to avoid merge conflicts on the JS part.

@wilsonge please do not squash the pr to honor the work of the others.

### CODE REVIEW

**Comment by brianteeman** [comment]
Has this been checked by the a11y team?

**Comment by laoneo** [comment]
All accessibility and UX efforts are on hold till the back end template is done. Here you can find more information https://volunteers.joomla.org/teams/new-media-manager-team/reports/743-media-manager-status-meeting-8-february-2018

**Comment by brianteeman** [comment]
Can you fix the minor drone issues please and then this should be merged

**Comment by laoneo** [comment]
Thanks, drone is fixed.

File: plugins/media-action/crop/form/crop.xml [hunk 1]
```diff
@@ -57,16 +57,28 @@
...
 			hiddenLabel="true"
-			class="btn-group btn-group-ratio"
-			default=""
+			class="crop-aspect-ratio-options"
+			default="1.111"
 			>
-			<option value="1.7777777777777777">16 / 9</option>
-			<option value="1.3333333333333333">4 / 3</option>
-			<option value="0.6666666666666666">2 / 3</option>
+			<option class="crop-aspect-ratio-option" value="1.111">Default aspect ratio</option>
```
Comment (Quy) [inline plugins/media-action/crop/form/crop.xml]: Convert to language string.

File: plugins/media-action/rotate/form/rotate.xml [hunk 2]
```diff
@@ -17,25 +17,28 @@
...
-			type="range"
-			label=""
-			value="0"
+			type="number"
+			label="PLG_MEDIA-ACTION_ROTATE_PARAM_ANGLE"
 			min="0"
 			max="360"
-			labelclass="d-none"
+			step="1"
+			value="0"
```
Comment (Quy) [inline plugins/media-action/rotate/form/rotate.xml]: Change to `default`

**Comment by laoneo** [comment]
Thanks @Quy! Done.

**Pull request merged**